views/admin/users.erb

```diff
@@ -156,5 +230,15 @@
                 $('#show-toolbox').show();
             });
         });
+        // if the users per page changes then we want to go back to the first page and begin showing the new number of users per page
```

Comment:
Remove this comment about users per page since pagination was removed

Reply:
done